### PR TITLE
feat: format and round echart tooltip

### PR DESCRIPTION
**Pull request by rephus**
### Pre requirements:
- [x] I have followed (at least) the [PR section of the contributing guide](https://github.com/lightdash/lightdash/blob/main/.github/CONTRIBUTING.md#sending-a-pull-request).

### Reference:
Closes: #1727

### Description:
Format and round numbers in tooltips 

### Preview:

image

image

image

image
Screenshot from 2022-05-09 09-47-47

### Scope of the changes (select all that apply):
- [x] Frontend  
- [ ] Backend  
- [ ] Common  
- [ ] E2E tests  
- [ ] Docs  
- [ ] CI/CD

### CODE REVIEW

File: packages/frontend/src/hooks/useEcharts.ts [hunk 1]
```diff
@@ -119,21 +119,38 @@ const getFormatterValue = (
...
+        const field = getFields(explore).find(
+            (item) => fieldId(item) === yField,
+        );
+
         if (Array.isArray(rawValue)) {
             const xValue = getFormatterValue(
                 rawValue[0],
                 xField,
                 getDimensions(explore),
             );
```
Comment (ZeRego) [inline packages/frontend/src/hooks/useEcharts.ts:131]: Might be less common but we need to format this value as well when we pivot.

**Review by ZeRego — CHANGES_REQUESTED** [review]

File: packages/frontend/src/hooks/useEcharts.ts (hunk 1 above)
Comment (rephus) [reply packages/frontend/src/hooks/useEcharts.ts:131]: Do you know how to replicate this ? I've already tried a few combinations with pivots and I don't get anything

Comment (ZeRego) [reply packages/frontend/src/hooks/useEcharts.ts:131]: Using scatter chart, the tooltip shows both x and y values.
https://lightdash-pr-2027.onrender.com/projects/3675b69e-8324-4110-bdca-059031aa8da3/tables/orders?create_saved_chart_version=%7B%22tableName%22%3A%22orders%22%2C%22metricQuery%22%3A%7B%22dimensions%22%3A%5B%22orders_is_completed%22%5D%2C%22metrics%22%3A%5B%22orders_average_order_size%22%2C%22orders_total_order_amount%22%5D%2C%22filters%22%3A%7B%7D%2C%22sorts%22%3A%5B%7B%22fieldId%22%3A%22orders_average_order_size%22%2C%22descending%22%3Atrue%7D%5D%2C%22limit%22%3A500%2C%22tableCalculations%22%3A%5B%5D%2C%22additionalMetrics%22%3A%5B%5D%7D%2C%22tableConfig%22%3A%7B%22columnOrder%22%3A%5B%22orders_average_order_size%22%2C%22orders_total_order_amount%22%2C%22orders_is_completed%22%5D%7D%2C%22chartConfig%22%3A%7B%22type%22%3A%22cartesian%22%2C%22config%22%3A%7B%22layout%22%3A%7B%22xField%22%3A%22orders_average_order_size%22%2C%22yField%22%3A%5B%22orders_total_order_amount%22%5D%2C%22flipAxes%22%3Afalse%7D%2C%22eChartsConfig%22%3A%7B%22series%22%3A%5B%7B%22encode%22%3A%7B%22xRef%22%3A%7B%22field%22%3A%22orders_average_order_size%22%7D%2C%22yRef%22%3A%7B%22field%22%3A%22orders_total_order_amount%22%7D%7D%2C%22type%22%3A%22scatter%22%7D%5D%7D%7D%7D%7D

Comment (rephus) [reply packages/frontend/src/hooks/useEcharts.ts:131]: Great, thanks. I fixed the XAxis, but I can see that the `key` for the tooltip still shows the unformatted value, however this is not handled in this method. Is this in scope for this ticket ? 
image

**Review by ZeRego — APPROVED** [review]

**Pull request merged**